src/main/java/nextstep/courses/domain/Price.java

```diff
@@ -13,15 +15,28 @@ public Price(int price) {
 
     private void inputValidation(int price) {
         if (price < 0) {
-            throw new IllegalArgumentException(String.format(가격은_D_원_이상이어야_합니다, MINIMUM_PRICE));
+            throw new IllegalArgumentException(String.format(MESSAGE_INVALID_MINIMUM_PRICE, MINIMUM_PRICE));
         }
     }
 
     public int price() {
         return price;
     }
 
-    public boolean samePrice(int price){
-        return this.price == price;
+    public boolean samePrice(Price price){
+        return this.equals(price);
```

Comment:
```suggestion
    public boolean samePrice(Price other){
        return price == other.price
```

이렇게는 어떨까요?
`equals` 도 좋지만, `equals`는 공통함수로 많이 쓰이기 때문에 `samePrice`를 도출했다면 직접 상태를 검증하는것도 좋을것 같아서요 🤔